### DISCUSSION TITLE
Is there a way to decrease the toc title font size or increase the toc column width?

### DISCUSSION THREAD

**Discussion by slerman12** [pradyunsg/furo#694] [Potential Issue]
Is there a way to decrease the toc title font size or increase the toc column width?

**Comment by Eric-Arellano** [comment]
Hey, you can customize CSS however you want with a custom CSS file, as described here: https://pradyunsg.me/furo/customisation/injecting/.

**Comment by slerman12** [comment]
Can you send a way to do it without modifying the css file?

**Reply by Eric-Arellano** [reply]
There is no other way to do it. It would not make sense for `conf.py` to allow you to customize arbitrary styling questions like that via Python config; that would result in Furo being an unmaintainable mess.

CSS is the way to make the change you want.

**Reply by slerman12** [reply]
Would it be sufficient to

html_style = 'css/yourtheme.css'

and then

just add a yourtheme.css with one parameter for the toc column title font?

What would that parameter be?

Thanks

**Reply by slerman12** [reply]
I set up the

templates_path = ['_templates']

(described here https://pradyunsg.me/furo/customisation/sidebar/)

 I have a _templates/sidebar directory structure with a file called title.html. Now what do I do?

**Reply by slerman12** [reply]
I know nothing about css or html!
